scripts/install_ebpf.psm1

```diff
@@ -296,11 +296,10 @@ function Install-eBPFComponents
     }
 
 
-    # Copy ebpfapi.dll from installation directory to current directory.
-    # This ensures latest version of ebpfapi is used by the tests.
-    $ebpfApiDllPath = Join-Path $env:ProgramFiles "ebpf-for-windows\ebpfapi.dll"
-    Write-Log("Copying ebpfapi.dll from '$ebpfApiDllPath' to '$pwd'...")
-    Copy-Item -Path $ebpfApiDllPath -Destination $pwd -Force
+    # Refresh Path so EbpfApi.dll can be found.
+    $machinepath = [system.environment]::getenvironmentvariable("path", [system.environmentvariabletarget]::machine)
+    $userpath = [system.environment]::getenvironmentvariable("path", [system.environmentvariabletarget]::user)
+    $env:path = $machinepath + ";" + $userpath
```

Comment:
I recently copied and pasted this pattern into a Developer PowerShell for VS2022 and found it broke all the tools implicitly added to its path. I wonder if there's anything we can do to preserve those paths?

Reply:
Looks like it doesn't matter in this scenario, which is PS remoting into a VM.